src/decorator.js

```diff
@@ -14,10 +14,17 @@ const createDecorator = (...calculations: Calculation[]): Decorator => (
           const next = getIn(values, field)
           const previous = getIn(previousValues, field)
           if (next !== previous) {
-            Object.keys(updates).forEach(destField => {
-              const update = updates[destField]
-              form.change(destField, update(next, values))
-            })
+            if (updates instanceof Function) {
```

Comment:
I think [`typeof > instanceof`](https://stackoverflow.com/questions/899574/which-is-best-to-use-typeof-or-instanceof)?